lang/en.json

```
@@ -180,6 +180,8 @@
 },
 
 "DND5E.ACTIVITY": {
+  "Create": "Create Activity",
+  "Delete": "Delete Activity",
```

For some other localizations recently I've been putting these in an an Action object:

```suggestion
  "Action": {
    "Create": "Create Activity",
    "Delete": "Delete Activity"
  },
```